src/test/java/com/jcabi/github/RtUserTest.java

```diff
@@ -475,6 +478,37 @@ public void notifications() throws Exception {
         );
     }
 
+    /**
+     * markAsRead() should complete successfully if response code is 205.
+     * @throws Exception Thrown in case of error.
+     * @todo #1304:30min Un-ignore this test and
```

Comment:
@llorllale I don't think it is the right direction. The Mk* classes are not made for us necessarily, but for the users of the library, who want to substitute the real api with the mock version... so we shouldn't alter it in such a way that it fits our tests, I believe (but maybe I'm wrong). 

Instead, see how other unit tests are written, they start an in-memory HTTP server (with MkGrizzly, from jcabi-http), instruct it what responses to give to any URI and then you can pass a custom ``Request`` to ``RtGithub`` (it is very similar to what we do in docker-java-api). 

So, I would say change this puzzle to what I said above (write tests with MkGrizzly mock server). WDYT?

Reply:
@amihaiemil ok